File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.java
class Solution {
    public int maxProduct(int[] nums) {
        PriorityQueue<Integer>pq=new PriorityQueue<>(Collections.reverseOrder());
        for(int i:nums){
            pq.offer(i);
        }
        int x=pq.poll();
        int y=pq.poll();
        return (x-1)*(y-1);
    }
}

File: 1464-maximum-product-of-two-elements-in-an-array/1464-maximum-product-of-two-elements-in-an-array.py
class Solution:
    def maxProduct(self, nums: List[int]) -> int:
        pq=[-i for i in nums]
        heapq.heapify(pq)
        x=-heapq.heappop(pq)
        y=-heapq.heappop(pq)
        return (x-1)*(y-1)